src/App.jsx

```diff
@@ -1,44 +1,37 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
 
+import { Tabs } from './components/Tabs/Tabs';
+
 export const tabs = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">Selected tab is Tab 1</h1>
-
-    <div data-cy="TabsComponent">
-      <div className="tabs is-boxed">
-        <ul>
-          <li className="is-active" data-cy="Tab">
-            <a href="#tab-1" data-cy="TabLink">
-              Tab 1
-            </a>
-          </li>
+export const App = () => {
+  const [activeTab, setActiveTab] = useState('tab-1');
+  const activeTabContent = tabs.find(tab => tab.id === activeTab);
 
-          <li data-cy="Tab">
-            <a href="#tab-2" data-cy="TabLink">
-              Tab 2
-            </a>
-          </li>
-
-          <li data-cy="Tab">
-            <a href="#tab-3" data-cy="TabLink">
-              Tab 3
-            </a>
-          </li>
-        </ul>
-      </div>
+  const handleTabClick = tabId => {
+    setActiveTab(tabId);
+  };
 
-      <div className="block" data-cy="TabContent">
-        Some text 1
+  return (
+    <div className="section">
+      <h1 className="title">Selected tab is {activeTabContent.title}</h1>
+      <div data-cy="TabsComponent">
+        <Tabs
+          tabs={tabs}
+          activeTab={activeTab}
```

Comment:
The prop `activeTab` should be renamed to `activeTabId` when passing it to the `Tabs` component, as per the task description: "The `App` should communicate with the `Tabs` component using the following props: `activeTabId` - the `id` of the tab that should be active."